utils/pinprocfw/pinprocfw.cpp

```diff
@@ -2134,13 +2136,10 @@ int processFile()
             endClock    = clock();
             fclose( in );
 
-            // Destroy the P-ROC device handle:
-            PRDelete(proc);
-            proc = kPRHandleInvalid;
             return iErrorCode;
```

Comment:
I moved the `PRDelete()` so it's always called, even if there's an error parsing the `.p-roc` file.